copy_and_nstrip_dll.ps1

```diff
@@ -31,6 +31,14 @@ else {
 $scriptDir = Split-Path -Parent -Path $MyInvocation.MyCommand.Definition
 $managedLibsFolder = $scriptDir + "\ManagedLibs"
 
+if (!(Test-Path $managedLibsFolder)) {
+    New-Item -ItemType Directory -Path $managedLibsFolder
+    Write-Output "ManagedLibs folder created successfully."
+}
+else {
+    Write-Output "ManagedLibs folder already exists."
+}

```

Comment:
Can you remove this else? I feel like the normal case is the ManagedLibs existing already. So there's really no point in notifying that it already exists.

Reply:
fixed in 08bece977d5ccebf7c10cb51e021e366b17fb71a